MediaBrowser.Providers/MediaInfo/FFProbeVideoInfo.cs

```diff
@@ -558,5 +579,31 @@ private ChapterInfo[] CreateDummyChapters(Video video)
 
             return chapters;
         }
+
+        private string[] FetchFromDvdLib(Video item)
+        {
+            var path = item.Path;
+            var dvd = new Dvd(path);
+
+            var primaryTitle = dvd.Titles.OrderByDescending(GetRuntime).FirstOrDefault();
+
+            byte? titleNumber = null;
+
+            if (primaryTitle is not null)
+            {
+                titleNumber = primaryTitle.VideoTitleSetNumber;
+                item.RunTimeTicks = GetRuntime(primaryTitle);
+            }
+
+            return _mediaEncoder.GetPrimaryPlaylistVobFiles(item.Path, titleNumber).ToArray();
+        }
+
+        private long GetRuntime(Title title)
+        {
+            return title.ProgramChains
+                    .Select(i => (TimeSpan)i.PlaybackTime)
+                    .Select(i => i.Ticks)
+                    .Sum();
```

Comment:
Pretty sure this can be a one-liner `title.ProgramChains.Sum(i => ((TimeSpan)i.PlaybackTime).Ticks);`